Source/Core/DiscIO/VolumeWiiCrypted.cpp

```diff
@@ -36,6 +36,12 @@ CVolumeWiiCrypted::CVolumeWiiCrypted(std::unique_ptr<IBlobReader> reader)
 {
   _assert_(m_pReader);
 
+  if (m_pReader->ReadSwapped<u32>(0x60) != u32(0))
```

Comment:
`ReadSwapped` returns `std::optional<u32>`, which can't be compared to an `int`.